src/CosmosDB/CosmosDB/Models/PSDatabaseAccount.cs

```diff
@@ -155,5 +156,9 @@ public PSDatabaseAccount(DatabaseAccountGetResults databaseAccountGetResults)
         //     Gets or sets Whether or not public endpoint access is allowed for this server.
         //     Possible values include: 'Enabled', 'Disabled'
         public string PublicNetworkAccess { get; set; }
+        //
+        // Summary:
+        //     Gets or sets list of Private Endpoint Connections configured for the Cosmos DB account.
+        public IList<PrivateEndpointConnection> PrivateEndpointConnections { get; set; }
```

Comment:
Is this not in CreateUpdateParamters object for DatabaseAccount ??

Reply:
It is only in get result.